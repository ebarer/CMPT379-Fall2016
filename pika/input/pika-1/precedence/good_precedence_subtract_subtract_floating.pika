exec {

  print 1.2 - 9.6 - 4.7, _n_.
  print 3.4 - 1.7 - 7.4, _n_.
  print 3.3 - 3.9 - 8.2, _n_.
  print 2.7 - 9.3 - 8.9, _n_.
  print 5.0 - 2.0 - 6.0, _n_.
  print 7.9 - 7.1 - 7.3, _n_.
  print 9.9 - 6.2 - 7.1, _n_.
  print 4.5 - 9.3 - 9.1, _n_.
  print 4.7 - 8.2 - 4.3, _n_.
  print 3.8 - 5.0 - 1.3, _n_.
  print 4.4 - 7.7 - 8.4, _n_.
  print 7.6 - 8.1 - 7.3, _n_.

  print 4.0 - 3.7 - 6.6, _n_.
  print 5.3 - 6.4 - 9.5, _n_.
  print 8.1 - 6.7 - 8.5, _n_.
  print 4.9 - 2.1 - 2.9, _n_.
  print 7.2 - 3.0 - 7.2, _n_.
  print 8.3 - 5.3 - 9.5, _n_.
  print 9.4 - 3.9 - 4.7, _n_.
  print 8.4 - 4.4 - 6.4, _n_.
  print 8.8 - 5.3 - 6.4, _n_.
  print 9.6 - 6.0 - 7.6, _n_.
  print 2.6 - 8.0 - 3.2, _n_.
  print 2.8 - 3.1 - 8.4, _n_.

  print 1.3 - 7.4 - 4.9 - 9.5, _n_.
  print 2.1 - 2.2 - 8.3 - 5.3, _n_.
  print 2.0 - 4.6 - 7.4 - 5.2, _n_.
  print 7.7 - 1.1 - 5.6 - 3.3, _n_.
  print 4.9 - 9.4 - 3.4 - 3.8, _n_.
  print 4.3 - 7.3 - 6.2 - 4.2, _n_.
  print 5.9 - 3.9 - 3.8 - 4.3, _n_.
  print 1.8 - 8.8 - 9.0 - 2.2, _n_.
  print 1.7 - 8.5 - 9.5 - 7.1, _n_.
  print 8.1 - 6.2 - 9.9 - 5.8, _n_.
  print 1.4 - 1.1 - 9.0 - 7.2, _n_.
  print 7.9 - 3.5 - 8.0 - 4.8, _n_.

  print 3.7 - 8.8 - 5.5 - 2.2, _n_.
  print 4.9 - 9.3 - 3.3 - 2.3, _n_.
  print 5.5 - 5.9 - 1.5 - 4.2, _n_.
  print 5.6 - 9.5 - 5.9 - 5.5, _n_.
  print 8.6 - 4.6 - 5.0 - 9.8, _n_.
  print 2.4 - 7.9 - 8.1 - 8.4, _n_.
  print 2.0 - 5.0 - 4.9 - 5.1, _n_.
  print 2.4 - 8.0 - 3.5 - 6.1, _n_.
  print 8.7 - 5.8 - 8.5 - 1.3, _n_.
  print 5.8 - 5.1 - 7.3 - 5.5, _n_.
  print 7.0 - 6.5 - 1.9 - 3.2, _n_.
  print 4.1 - 7.1 - 5.4 - 3.3, _n_.

}

