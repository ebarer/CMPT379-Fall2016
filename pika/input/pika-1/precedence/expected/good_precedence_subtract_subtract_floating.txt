-13.1
-5.7
-8.8
-15.5
-3
-6.5
-3.4
-13.9
-7.8
-2.5
-11.7
-7.8
-6.3
-10.6
-7.1
-0.1
-3
-6.5
0.8
-2.4
-2.9
-4
-8.6
-8.7
-20.5
-13.7
-15.2
-2.3
-11.7
-13.4
-6.1
-18.2
-23.4
-13.8
-15.9
-8.4
-12.8
-10
-6.1
-15.3
-10.8
-22
-13
-15.2
-6.9
-12.1
-4.6
-11.7
